.github/workflows/sanity.yaml

```diff
@@ -0,0 +1,56 @@
+# This runs on
+# - Push to main
+# - PRs
+name: test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  ACUVITY_TOKEN: ${{ secrets.ACUVITY_TOKEN }}
+
+jobs:
+  sanity:
+    name: Sanity
+    runs-on: ubuntu-latest
+    concurrency:
+      group: matrix-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v2
+      - uses: denoland/setup-deno@v2
```

Comment:
nice! they have GitHub actions for that!

Reply:
yup, I was trying the curl cmds but then googled it and boom 1st link. :D